spec/ralyxa/response_entities/directives/audio/stream_spec.rb

```diff
@@ -0,0 +1,69 @@
+require_relative '../../../../spec_helper'
+
+RSpec.describe Ralyxa::ResponseEntities::Directives::Audio::Stream do
+  describe '#initialize' do
+    context 'with only the required values' do
+      subject(:stream) { described_class.new('http://localhost', 'my-token') }
+
+      it 'sets the expected instance variables' do
+        expect(stream.instance_variable_get(:@url)).to eq('http://localhost')
+        expect(stream.instance_variable_get(:@token)).to eq('my-token')
+        expect(stream.instance_variable_get(:@offset_in_milliseconds)).to eq(0)
+        expect(stream.instance_variable_get(:@expected_previous_token)).to eq(nil)
+      end
+    end
```

Comment:
Ditto here (I avoid testing initializers explicitly, prefer to test behaviour dependent on them at a different level of abstraction to the implementation)